src/components/TimeLineText.css

```diff
@@ -0,0 +1,22 @@
+.wui-timeline-container {
+  display: flex;
+  justify-content: space-between;
+  text-align: justify;
+}
+
+.wui-timeline-container div {
+  width: 80%;
+}
+
+.wui-timeline-container span {
+  align-self: center;
+}
+
+.wui-hr {
+  display: flex;
+  flex: 1;
+}
+
+hr {
```

Comment:
I think if this is needed it should also be 'name-spaced' otherwise it would affect any `hr` in client projects

Reply:
Done. Thanks.